lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -331,6 +331,10 @@ class CategoryRenderer {
     wrapper.href = `#${category.id}`;
     wrapper.classList.add(`lh-gauge__wrapper--${Util.calculateRating(category.score)}`);
 
+    if (!['performance', 'accessibility', 'best-practices', 'seo', 'pwa'].includes(category.id)) {
```

Comment:
@brendankenny is this the best way to detect plugins?  Seems _okay_ to me, but the code smell is not great.

Reply:
let's at least make it a util function :)

Reply:
also plugin category ids should have their name match the module that was passed in `--plugins` so we could check that instead?

I suppose it's a question of if we want to flag all plugins with this icon or flag all non-core groups with this icon.

Reply:
> let's at least make it a util function :)

sure

> also plugin category ids should have their name match the module that was passed in --plugins so we could check that instead?

how would I access that from this function?